### PR TITLE
Images were added to img/etherscan-voting, and the etherscan_voting.m…

### PR DESCRIPTION
**Pull request by JennyKop**
Images were added to img/etherscan-voting, and the etherscan_voting.md was changed to a new one referring to the new voting contract, after vote 178 enacted

### CODE REVIEW

**Review by rkolpakov — APPROVED** [review]
LGTM

File: docs/guides/etherscan-voting.md
Comment (tamtamchik) [inline docs/guides/etherscan-voting.md:66]: Assuming the initial steps have been completed, this step is redundant. It is also not present in all other flows, so I don't see a reason why it is present here. Maybe remove it?

**Review by tamtamchik — CHANGES_REQUESTED** [review]
There is an open question with a different address for the 'Aragon Voting' contract. It needs to be settled before approval.

The other comments are just suggestions; they are nice to have but not essential or blocking.

File: docs/guides/etherscan-voting.md
Comment (tamtamchik) [reply docs/guides/etherscan-voting.md:8]: It looks like 0xf16514 is an implementation address, probably just a typo.

Comment (JennyKop) [reply docs/guides/etherscan-voting.md]: Correct, thanks for checking! Changed.

Comment (JennyKop) [reply docs/guides/etherscan-voting.md]: Changed, thanks for sharing

File: docs/guides/etherscan-voting.md
Comment (JennyKop) [reply docs/guides/etherscan-voting.md:15]: Make sence

**Review by tamtamchik — APPROVED** [review]
Approved! :)

**Pull request merged**
